cmd/antrea-controller/controller.go

```diff
@@ -100,23 +103,50 @@ func run(o *Options) error {
 
 	go apiServer.GenericAPIServer.PrepareRun().Run(stopCh)
 
+	if o.config.EnablePrometheusMetrics {
+		go initializePrometheusMetrics()
+	}
+
 	<-stopCh
 	klog.Info("Stopping Antrea controller")
 	return nil
 }
 
+// Initialize Prometheus metrics collection.
+func initializePrometheusMetrics() {
+
+	nodeName, err := env.GetNodeName()
+	if err != nil {
+		klog.Errorf("Failed to retrieve controller K8S node name, %v", err)
```

Comment:
```suggestion
		klog.Errorf("Failed to retrieve controller K8S node name: %v", err)
```
Could you unify log style of this PR in this way? It's the most common go error log style.

Reply:
Done